### ISSUE TITLE
everyone who uses VT-2 is dumb

### ISSUE THREAD

**Issue by OddbyteWasTaken** [dnschneid/crouton#5051]
here is how to [use sudo in crosh](https://github.com/OddbyteWasTaken/howto-use-sudo-in-crosh/tree/main). Follow any tutorial you like

**Comment by OddbyteWasTaken** [comment]
main benefit for me is that you can copy paste sudo commands into there

**Comment by grumpyman12601** [comment]
Getting this error:
ssh: connect to host localhost port 22: Connection refused

Do you have read/write protection disabled? If so that is probably my issue, still waiting on a breakout board to get delivered.

**Comment by DennisLfromGA** [comment]
It also appears that you need to assign both root & chronos passwords. 🤔

**Comment by OddbyteWasTaken** [comment]
i have edited the repo with new instructions @DennisLfromGA @grumpyman12601

**Comment by OddbyteWasTaken** [comment]
new instructions do not require any firmware mods or WP disabled. you only need access to the VT2 terminal and a wifi connection (to install netcat)

**Comment by OddbyteWasTaken** [comment]
this new method does not require any passwords or opening your ssh port, so either way this method is safer

**Comment by DennisLfromGA** [comment]
@OddbyteWasTaken,

ssh'ing into it from the penguin container is easier for me, no chromebrew required.  

You can even set up passwordless ssh too by adding the penguin id_rsa.pub file to the chronos authorized_keys file.

-**DennisLfromGA**

**Comment by OddbyteWasTaken** [comment]
> @OddbyteWasTaken,
> 
> `ssh`'ing into it from the penguin container is easier for me, no `chromebrew` required.
> 
> You can even set up passwordless `ssh` too by adding the penguin `id_rsa.pub` file to the chronos `authorized_keys` file.
> 
> -**DennisLfromGA**

while true, installing chromebrew is useful anyways, and i dont like crosstini :P
also, for some reason sshd broke for my chromebook, and i dont feel like powerwashing

**Comment by DennisLfromGA** [comment]
@OddbyteWasTaken,

It's nice to have options.  

`sshd` broke on mine too but I can launch it manually using [the opensshd instructions](https://chromium.googlesource.com/chromiumos/overlays/chromiumos-overlay/+/master/chromeos-base/chromeos-sshd-init/files/openssh-server.conf.README#:~:text=Manually%20start%20sshd.) when needed.

-**DennisLfromGA**

**Comment by OddbyteWasTaken** [comment]
@DennisLfromGA
The tutorial has been updated with the crosstini methods

**Comment by CroutonIsFun** [comment]
Your title is disingenuous and insulting. Since "everyone" who uses VT-2 is dumb, this includes you. Also, this does not allow you to use crosh: it opens up a new virtual terminal, which is xfce4-terminal.

Still, it's a clever approach, and shows that there is not just one way to solve a problem.

**Comment by Echote1** [comment]
like grah keep it a stack

**Comment by OddbyteWasTaken** [comment]
> Your title is disingenuous and insulting. Since "everyone" who uses VT-2 is dumb, this includes you. Also, this does not allow you to use crosh: it opens up a new virtual terminal, which is xfce4-terminal.
> 
> Still, it's a clever approach, and shows that there is not just one way to solve a problem.

Yes, i agree that i am insulting. however, it is a joke, so chill.

**Comment by CroutonIsFun** [comment]
Lolol will do

**Comment by CroutonIsFun** [comment]
I was disappointed that I could not launch crouton from xfce4-terminal. I still have to run ssh inside my throat, then ssh from the crosh shell, and only then launch the desktop.

Even though xfce4-terminal is a graphical program, it's running inside VT-2, so you can't launch a graphical session.

**Comment by hattmall1** [comment]
Do any of these methods allow you to launch crouton? If so would it be possible to autostart any of these and then launch crouton. I was previously able to just tun on my computer and have it open in the xfce4. Now it's a major pita.

**Comment by CroutonIsFun** [comment]
No, although a few things can be automated.

**Comment by justapcgamer** [comment]
Method 5 is super easy

**Comment by oddbyte** [comment]
> Method 5 is super easy

well it is also the one with the highest risk of bricking your device
